controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1190,12 +1190,15 @@ bool master_thread::handle_intel_slave_join(
     Socket *sd, std::shared_ptr<wfa_map::tlvApRadioBasicCapabilities> radio_caps,
     ieee1905_1::CmduMessageRx &cmdu_rx, ieee1905_1::CmduMessageTx &cmdu_tx)
 {
-    auto tlv_vs = cmdu_tx.add_vs_tlv(ieee1905_1::tlvVendorSpecific::eVendorOUI::OUI_INTEL);
-    if (!tlv_vs) {
+    // Prepare outcoming response vs tlv
+    auto join_response_vs_tlv =
```

Comment:
Again, it's not a join_response_vs_tlv that you get here, it's just the join_response.

Reply:
will fix.